lib/ui/pointer.dart

```diff
@@ -111,6 +115,10 @@ class PointerData {
   /// Unique identifier for the pointing device, reused across interactions.
   final int device;
 
+  /// Unique identifier for the pointer, not reused. Changes for each new
+  /// pointer down event.
+  final int pointer;
```

Comment:
Please elaborate on the purpose of this field in the documentation. How would a caller use the same? Maybe name this `pointer_identifier`?